src/main/webapp/app/atlas/manage/learning-paths-table/learning-paths-table.component.html

```diff
@@ -56,6 +56,9 @@ <h5 class="m-0 col-md-auto p-0" jhiTranslate="artemisApp.learningPathManagement.
                     }
                 </tbody>
             </table>
+            @if (averageProgress() !== undefined) {
+                <div class="mt-3"><strong jhiTranslate="artemisApp.learningPathManagement.averageProgress"></strong>: {{ averageProgress() }}%</div>
```

Comment:
are you on purpose writing averageProgress() as fallback? Maybe just use the english translation Average Progress as fallback